src/features/defi/providers/fox-farming/components/FoxFarmingManager/Overview/Claim/ClaimConfirm.tsx

```diff
@@ -97,6 +131,15 @@ export const ClaimConfirm = ({
         chainId,
         contractAddress,
       })
+      trackOpportunityEvent(
+        MixPanelEvents.ClaimConfirm,
+        {
+          opportunity,
+          fiatAmounts: [claimFiatAmount],
+          cryptoAmounts: [{ assetId: asset?.assetId, amountCryptoHuman: amount }],
```

Comment:
Asset should be defined after l.119
```suggestion
          cryptoAmounts: [{ assetId: asset.assetId, amountCryptoHuman: amount }],
```